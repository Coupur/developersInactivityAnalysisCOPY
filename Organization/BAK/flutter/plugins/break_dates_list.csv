mit-mit,2017-04-19/2017-04-24,2017-04-24/2017-05-09,2017-05-09/2017-05-16,2017-05-16/2017-05-23,2017-05-23/2017-05-31,2017-05-31/2017-06-08,2017-06-08/2017-06-15,2017-06-15/2017-07-26,2017-07-26/2017-08-15,2017-08-16/2017-08-24,2017-08-24/2017-09-05,2017-09-05/2017-09-14,2017-09-14/2017-10-11,2017-10-11/2017-10-13,2017-10-13/2017-10-24,2017-10-24/2017-12-12,2017-12-12/2018-01-11,2018-01-12/2018-02-01,2018-02-01/2018-05-06,2018-05-07/2019-01-06,2019-01-06/2019-06-13
szakarias,2017-04-21/2017-04-24,2017-04-24/2017-04-27,2017-04-27/2017-05-01,2017-05-01/2017-05-03,2017-05-05/2017-05-08,2017-05-11/2017-05-15,2017-05-15/2017-05-17,2017-05-17/2017-06-15,2017-06-16/2018-02-15,2018-02-15/2018-04-03,2018-04-04/2018-04-12,2018-04-12/2018-04-20,2018-04-20/2018-04-23,2018-04-24/2018-04-30,2018-05-01/2018-05-03,2018-05-03/2018-05-07,2018-05-07/2018-05-13,2018-05-14/2018-05-24,2018-05-24/2018-07-03
collinjackson,2017-05-03/2017-05-05,2017-05-05/2017-05-11,2017-05-11/2017-05-16,2017-05-16/2017-05-18,2017-05-18/2017-05-23,2017-05-23/2017-06-03,2017-06-03/2017-06-05,2017-06-05/2017-06-14,2017-06-14/2017-06-16,2017-06-16/2017-06-20,2017-06-20/2017-06-22,2017-06-22/2017-07-07,2017-07-07/2017-07-17,2017-07-17/2017-08-30,2017-08-30/2017-09-11,2017-09-12/2017-10-03,2017-10-03/2017-10-17,2017-10-17/2017-10-27,2017-10-27/2017-11-28,2017-11-28/2017-12-05,2017-12-05/2017-12-18,2017-12-18/2017-12-20,2017-12-20/2018-01-20,2018-01-20/2018-04-02,2018-04-02/2018-04-05,2018-04-05/2018-04-09,2018-04-10/2018-04-12,2018-04-12/2018-05-07,2018-05-07/2018-05-21,2018-05-21/2018-11-21,2018-11-21/2018-11-27,2018-11-27/2019-02-11,2019-02-11/2019-02-13,2019-02-13/2019-02-21,2019-02-21/2019-03-06,2019-03-06/2019-03-14,2019-03-15/2019-03-18,2019-03-18/2019-03-21,2019-03-21/2019-04-05,2019-04-05/2019-04-08,2019-04-08/2019-04-10,2019-04-10/2019-04-17,2019-04-19/2019-04-22,2019-04-23/2019-04-29,2019-04-29/2019-05-01,2019-05-01/2019-05-03,2019-05-03/2019-05-06,2019-05-08/2019-05-14,2019-05-15/2019-05-29,2019-05-30/2019-06-05,2019-06-06/2019-06-11,2019-06-13/2019-06-15
xster,2017-05-05/2017-05-08,2017-05-10/2017-07-07,2017-07-07/2017-10-18,2017-10-18/2018-01-12,2018-01-12/2018-01-30,2018-01-30/2018-02-02,2018-02-02/2018-03-20,2018-03-20/2018-03-29,2018-03-29/2018-04-09,2018-04-09/2018-12-11
goderbauer,2017-05-12/2017-05-16,2017-05-16/2017-05-24,2017-05-24/2017-05-30,2017-05-30/2017-06-05,2017-06-05/2017-06-08,2017-06-08/2017-06-16,2017-06-16/2017-06-20,2017-06-20/2017-06-22,2017-06-22/2017-07-06,2017-07-06/2017-07-17,2017-07-17/2017-07-25,2017-07-25/2017-09-25,2017-09-25/2017-10-20,2017-10-20/2017-12-14,2017-12-14/2018-05-01,2018-05-02/2018-06-01,2018-06-01/2018-08-15,2018-08-15/2019-03-08
natelust
efortuna,2017-05-16/2018-12-19
mehmetf,2017-05-18/2017-06-06,2017-06-06/2017-06-09,2017-06-09/2017-06-12,2017-06-12/2017-06-16,2017-06-16/2017-06-29,2017-06-29/2017-07-06,2017-07-07/2017-09-19,2017-09-19/2017-09-22,2017-09-22/2017-10-19,2017-10-19/2017-11-04,2017-11-06/2017-11-18,2017-11-18/2017-11-29,2017-11-29/2017-12-12,2017-12-12/2018-05-29,2018-05-29/2018-06-16,2018-06-16/2018-09-25,2018-09-27/2018-10-01,2018-10-01/2018-10-24,2018-10-24/2018-10-28,2018-10-28/2018-11-07,2018-11-07/2019-01-06,2019-01-06/2019-02-22,2019-02-22/2019-04-09,2019-04-09/2019-04-15,2019-04-15/2019-05-22,2019-05-23/2019-05-30
perlatus
branflake2267,2017-05-23/2017-05-30,2017-05-30/2017-06-01,2017-06-01/2017-06-05,2017-06-05/2017-06-12,2017-06-12/2017-06-19,2017-06-19/2017-08-17,2017-08-17/2017-09-25,2017-09-25/2017-10-03,2017-10-03/2017-11-28,2017-11-28/2018-04-11,2018-04-11/2018-04-17
LarkAscending
killermonk,2017-05-27/2018-05-14
arashbi,2017-05-29/2017-07-19
jmolins
tvolkert,2017-06-05/2017-06-07,2017-06-07/2017-06-12,2017-06-13/2017-06-16,2017-06-16/2017-06-20,2017-06-20/2017-06-26,2017-06-27/2017-06-29,2017-06-29/2017-07-05,2017-07-05/2017-07-14,2017-07-14/2017-09-29,2017-09-29/2017-12-02
sadilek
pulyaevskiy,2017-06-08/2017-11-15,2017-11-15/2017-11-28,2017-11-28/2018-01-05,2018-01-05/2018-01-17,2018-01-17/2018-03-05,2018-03-05/2018-04-23,2018-04-23/2018-04-29,2018-04-29/2018-09-05,2018-09-05/2018-11-09
gspencergoog,2017-06-09/2018-08-20,2018-08-21/2019-01-16
kevmoo,2017-06-13/2018-09-01
Skylled,2017-06-13/2017-10-27,2017-10-27/2017-11-28,2017-11-28/2018-01-03,2018-01-03/2018-01-13,2018-01-13/2018-01-20,2018-01-20/2018-03-23
yjbanov,2017-06-30/2017-08-15,2017-08-15/2018-09-26
lukef,2017-07-07/2018-02-08
PixelToast
HansMuller,2017-07-13/2018-01-03,2018-01-04/2018-03-20
sethladd,2017-07-13/2017-08-21,2017-08-21/2017-08-28,2017-08-28/2017-10-03
jxmorris12
BenSower,2017-07-24/2017-10-06,2017-10-06/2017-10-20
mravn-google,2017-07-29/2017-08-01,2017-08-01/2017-08-08,2017-08-08/2017-10-25,2017-10-26/2017-11-10,2017-11-11/2017-11-28,2017-11-29/2017-12-01,2017-12-03/2017-12-06,2017-12-06/2017-12-14,2017-12-15/2017-12-18,2017-12-20/2018-01-02,2018-01-02/2018-01-09,2018-01-12/2018-01-17,2018-01-18/2018-02-12,2018-02-12/2018-02-27,2018-03-01/2018-03-04,2018-03-04/2018-03-06,2018-03-06/2018-03-09,2018-03-09/2018-03-15,2018-03-15/2018-03-20,2018-03-21/2018-03-25,2018-03-26/2018-04-02,2018-04-02/2018-04-05,2018-04-05/2018-04-09,2018-04-11/2018-04-13,2018-04-13/2018-04-16,2018-04-16/2018-04-18,2018-04-18/2018-05-01,2018-05-01/2018-05-04,2018-05-04/2018-05-07,2018-05-08/2018-05-13,2018-05-15/2018-05-23,2018-05-23/2018-05-25,2018-05-25/2018-05-28,2018-05-28/2018-06-01,2018-06-01/2018-06-06,2018-06-07/2018-06-12,2018-06-12/2018-06-15,2018-06-15/2018-06-23
frederickcook
kroikie,2017-08-03/2017-10-17,2017-10-17/2017-11-06,2017-11-06/2018-01-04,2018-01-04/2018-02-01,2018-02-02/2018-02-07,2018-02-07/2018-04-09,2018-04-09/2018-04-30,2018-04-30/2018-05-07,2018-05-07/2018-06-13,2018-06-13/2018-06-21,2018-06-21/2018-06-24,2018-06-24/2018-07-11,2018-07-11/2018-07-16,2018-07-18/2018-07-20,2018-07-20/2018-08-30,2018-08-30/2018-09-05,2018-09-05/2018-10-05,2018-10-05/2018-10-09,2018-10-09/2018-10-11,2018-10-12/2018-10-18,2018-10-18/2018-11-08,2018-11-08/2018-11-10,2018-11-10/2019-03-20,2019-03-20/2019-03-26,2019-03-26/2019-04-05,2019-04-05/2019-04-08,2019-04-08/2019-05-14,2019-05-14/2019-05-29,2019-05-29/2019-05-31,2019-05-31/2019-06-04,2019-06-04/2019-06-13,2019-06-13/2019-06-17,2019-06-17/2019-06-19
mogol,2017-08-16/2017-10-17
pcomans
kykl
aler
SiarheiFedartsou,2017-10-03/2017-10-10
lejard-h
MarcinusX,2017-10-10/2018-01-13
xqwzts,2017-10-17/2019-05-30
jason-simmons,2017-10-24/2017-11-10,2017-11-10/2017-11-13,2017-11-13/2019-05-09,2019-05-09/2019-05-28,2019-05-28/2019-06-04
roughike,2017-10-27/2017-11-14,2017-11-14/2017-11-20,2017-11-20/2018-03-29,2018-03-29/2018-04-03,2018-04-03/2018-04-19,2018-04-19/2018-04-23
zanderso,2017-10-31/2017-11-14,2017-11-14/2017-12-04,2017-12-04/2017-12-08,2017-12-08/2017-12-15,2017-12-15/2017-12-19,2017-12-19/2018-03-20
devoncarew,2017-11-17/2018-04-04
maxatcn
sigurdm,2017-12-01/2017-12-04,2017-12-05/2017-12-07,2017-12-07/2017-12-19,2017-12-19/2018-01-09,2018-01-09/2018-01-11,2018-01-11/2018-01-30,2018-01-30/2018-02-13,2018-02-13/2018-04-16,2018-04-16/2018-05-15,2018-05-15/2018-06-07,2018-06-07/2019-04-04
rock3r
sebasbad
timsneath
alibitek,2018-01-04/2018-01-11
tlarsen7572
RedBrogdon,2018-01-04/2018-01-23,2018-01-23/2018-03-19
pedia,2018-01-12/2018-01-26
KevinTheGray
polbatllo
romaluca
charafau
briblanch
long1eu,2018-02-05/2019-05-14
jarrodsinclair
a-siva
aam
mraleph,2018-02-27/2018-03-02,2018-03-02/2018-03-12,2018-03-12/2018-03-14
abykov2
xtelinco
fkorotkov,2018-03-09/2018-03-12,2018-03-12/2018-03-14,2018-03-14/2018-03-16,2018-03-16/2018-04-03,2018-04-03/2018-06-26
jsoref
jonahwilliams,2018-03-16/2018-07-10,2018-07-10/2018-09-28
ypelud,2018-03-16/2018-03-28
rkowase
kurtisnelson
namanix
jakobr-google
shinriyo
victorbonnet
grandstaish
bparrishMines,2018-04-11/2018-04-16,2018-04-16/2018-05-04,2018-05-04/2018-05-25,2018-05-25/2018-05-29,2018-05-29/2018-06-06,2018-06-06/2018-06-12,2018-06-12/2018-06-14,2018-06-15/2018-06-20,2018-06-20/2018-06-22,2018-06-22/2018-06-25,2018-06-25/2018-06-28,2018-06-29/2018-07-16,2018-07-19/2018-07-24,2018-07-25/2018-08-17,2018-08-17/2018-08-23,2018-08-23/2018-09-05,2018-09-05/2018-09-21,2018-09-21/2018-09-29,2018-09-29/2018-10-01,2018-10-03/2018-10-10,2018-10-10/2018-10-16,2018-10-16/2018-10-31,2018-11-01/2018-11-08,2018-11-08/2018-11-13,2018-11-13/2018-11-16,2018-11-16/2019-01-03,2019-01-03/2019-01-28,2019-01-28/2019-01-30,2019-01-30/2019-02-11,2019-02-14/2019-02-19,2019-02-20/2019-03-04,2019-03-04/2019-03-14,2019-03-14/2019-03-22,2019-03-22/2019-04-04,2019-04-04/2019-04-11,2019-04-11/2019-04-19,2019-04-19/2019-04-26,2019-04-26/2019-05-14,2019-05-14/2019-05-16,2019-05-16/2019-05-29,2019-05-30/2019-06-06,2019-06-06/2019-06-11,2019-06-11/2019-06-13,2019-06-13/2019-06-17
kuemit
luigi-agosti
sroddy,2018-05-02/2018-05-15
ihildebrandt
bjon,2018-05-02/2018-05-15
bizz84,2018-05-04/2019-04-05
imtoori,2018-05-04/2018-05-07
kmcnellis
pauldemarco,2018-05-07/2018-09-04
azihsoyn,2018-05-08/2018-07-13
floitschG
quentinleguennec,2018-05-11/2018-05-17,2018-05-17/2018-10-17,2018-10-17/2019-04-15
mtaliaf
kondasoft,2018-05-14/2018-06-08,2018-06-08/2018-10-17
najeira
jonasbark,2018-05-15/2018-06-07,2018-06-07/2018-06-15
mortenboye
haydenflinner
g123k
josephch405
bcko
skybur
AppleEducate
slightfoot,2018-06-05/2018-07-26,2018-07-26/2018-09-05,2018-09-05/2018-09-27,2018-09-27/2018-11-22
gerryhigh
johnhaitas
chinmaygarde
wreppun
gbaccetta
urbandove
IhorKlimov
cbenhagen,2018-06-26/2019-03-04,2019-03-04/2019-04-25
pyros2097
kwmt,2018-06-26/2019-02-21
Noble-Mushtak
Maleandr
ffeu,2018-07-25/2018-09-04,2018-09-04/2018-10-30,2018-10-30/2018-11-14,2018-11-14/2018-11-16,2018-11-16/2019-02-25
thmsdnnr
creativecreatorormaybenot,2018-07-26/2019-04-08,2019-04-08/2019-04-29,2019-04-29/2019-05-27,2019-05-28/2019-06-04,2019-06-04/2019-06-10,2019-06-10/2019-06-15
paolorotolo,2018-07-30/2018-08-22
a14n,2018-08-01/2018-08-03,2018-08-03/2018-09-12,2018-09-12/2018-10-08,2018-10-08/2018-10-16,2018-10-16/2018-10-19
MichaelRFairhurst
bkonyi,2018-08-07/2018-08-09,2018-08-09/2018-08-15,2018-08-15/2018-08-24,2018-08-24/2018-12-26,2018-12-26/2019-01-29,2019-01-29/2019-02-05,2019-02-05/2019-02-13,2019-02-13/2019-02-21,2019-02-21/2019-03-22,2019-03-22/2019-03-25
sir-boformer,2018-08-15/2018-08-28,2018-08-28/2018-08-30
Cretezy,2018-08-15/2018-11-06
nullp01nt3r
jiajiabingcheng
sestegra
fredriks
diegoveloper,2018-08-23/2018-12-25
lmNt
nichtverstehen,2018-08-27/2018-09-06,2018-09-06/2018-09-11
kdy1
amirh,2018-08-31/2018-09-05,2018-09-06/2018-10-16,2018-10-16/2018-11-10,2018-11-10/2018-11-20,2018-11-22/2018-11-26,2018-11-30/2018-12-03,2018-12-03/2018-12-06,2018-12-06/2018-12-08,2018-12-08/2018-12-20,2018-12-20/2019-01-04,2019-01-04/2019-01-10,2019-01-10/2019-01-12,2019-01-12/2019-01-23,2019-01-23/2019-01-31,2019-01-31/2019-02-04,2019-02-04/2019-02-14,2019-02-14/2019-02-22,2019-02-22/2019-02-25,2019-02-25/2019-02-27,2019-02-27/2019-03-09,2019-03-09/2019-03-14,2019-03-14/2019-03-19,2019-03-20/2019-03-23,2019-03-23/2019-03-26,2019-03-27/2019-03-29,2019-03-30/2019-04-02,2019-04-02/2019-05-15,2019-05-15/2019-05-22,2019-05-22/2019-05-28
nkmrh
westoque
mickykebe
bkayfitz-cara
DennisAlund,2018-09-07/2018-10-18
kmorkos
rydein
JamesMcIntosh
eva95678
samtstern
BWMuller
ko2ic,2018-10-03/2018-11-21,2018-11-21/2019-03-27,2019-03-27/2019-04-05
LucasParsy
duncanhoggan
kenzieschmoll
YAPPDEF
dnfield,2018-10-17/2018-11-03,2018-11-03/2018-11-10,2018-11-10/2018-11-19
huangminlinux
pinkfish
matthewtsmith
ksheremet,2018-10-30/2019-03-28
ryuta46
mklim,2018-11-05/2018-11-27,2018-11-28/2019-01-03,2019-01-03/2019-01-07,2019-01-07/2019-01-14,2019-01-16/2019-01-18,2019-01-18/2019-01-23,2019-01-25/2019-01-28,2019-01-28/2019-02-08,2019-02-08/2019-02-11,2019-02-11/2019-02-14,2019-02-15/2019-02-19,2019-02-19/2019-02-22,2019-02-22/2019-02-25,2019-02-26/2019-02-28,2019-02-28/2019-03-07,2019-03-08/2019-04-19,2019-04-19/2019-04-24,2019-04-24/2019-04-29,2019-04-29/2019-05-01,2019-05-03/2019-05-08,2019-05-08/2019-05-10
bashtian
filleduchaos
ahmedb
songoku1610
rockwotj
martflu
timtraversy,2018-11-20/2018-11-25,2018-11-25/2019-02-15
ened,2018-11-21/2018-11-23,2018-11-23/2019-02-22,2019-02-22/2019-03-13,2019-03-13/2019-03-22,2019-03-22/2019-04-15,2019-04-15/2019-05-13
lennonjesus
dguralnick
Rahiche
chadidi
alanrussian
akindone
kikuchy
YukiOya,2018-12-08/2019-06-13
Patte1808
wouterhardeman,2018-12-20/2019-01-03,2019-01-03/2019-01-31,2019-01-31/2019-05-31
matanlurey
bcollins19
kangwang1988,2019-01-14/2019-01-30,2019-01-30/2019-02-08
recastrodiaz
edman
davidair
iskakaushik,2019-02-07/2019-02-13,2019-02-13/2019-02-21,2019-02-21/2019-02-27,2019-02-28/2019-03-05,2019-03-06/2019-03-08,2019-03-08/2019-03-29,2019-03-29/2019-04-02,2019-04-04/2019-04-11,2019-04-11/2019-04-13,2019-04-13/2019-04-17,2019-04-17/2019-04-22,2019-04-23/2019-04-26,2019-04-26/2019-04-29,2019-04-29/2019-05-01,2019-05-01/2019-06-05,2019-06-05/2019-06-17
idealopamp,2019-02-07/2019-02-22
aoinakanishi
IAmSarthakVerma
cyanglaz,2019-02-22/2019-02-25,2019-02-26/2019-02-28,2019-02-28/2019-03-02,2019-03-02/2019-03-06,2019-03-06/2019-03-08,2019-03-08/2019-03-19,2019-03-19/2019-03-22,2019-03-22/2019-03-29,2019-03-29/2019-04-02,2019-04-04/2019-04-06,2019-04-06/2019-04-08,2019-04-12/2019-04-18,2019-04-19/2019-04-23,2019-04-23/2019-04-25,2019-04-26/2019-05-01,2019-05-01/2019-05-20,2019-05-20/2019-05-22,2019-05-22/2019-05-29,2019-05-31/2019-06-03,2019-06-04/2019-06-06,2019-06-06/2019-06-14
ivk1800,2019-02-21/2019-03-08,2019-03-08/2019-03-11,2019-03-12/2019-03-19,2019-03-19/2019-04-08,2019-04-09/2019-04-23,2019-04-23/2019-04-30
hirohisa
MasterCarl
sam-toon
andreidiaconu
cvolzke4,2019-02-28/2019-03-13
Salakar,2019-03-01/2019-06-19
rishab2113,2019-03-01/2019-06-18
gi097
warnyul
PonnamKarthik
Ehesp
p30arena,2019-03-06/2019-03-27
terryBaz
PaitoAnderson,2019-03-13/2019-04-12
vinayphadnis
dtq
hpoul
leoafarias
KaYBlitZ,2019-03-22/2019-04-17
gayanpathirage
ninnepinne
kushvatsa
dotdoom,2019-03-28/2019-04-10
tsuba3
thomas-stockx
alextekartik
jul1u5
DipanshKhandelwal
anaisbetts
RafaO
wwwdata
miguelpruivo
zaki50
sarbagyastha,2019-04-08/2019-05-31
yunyu
awazgyawali,2019-04-10/2019-05-06,2019-05-06/2019-05-08,2019-05-08/2019-05-18,2019-05-18/2019-06-06
LinusU,2019-04-12/2019-04-25,2019-04-25/2019-04-29,2019-04-29/2019-05-17
domesticmouse
yadaniyil
vargavince91
apm29
athornz
laramartin
apaatsio
matthew-carroll
iqbalmineraltown
matthewlloyd,2019-04-29/2019-05-02,2019-05-02/2019-05-06,2019-05-06/2019-05-21,2019-05-21/2019-05-30
mono0926
EdDT-Phi
jaumard
csobrinho
zerefati
lesnitsky
viniciuslrangel
Vitaly-V
topeomot2
axellebot
roeierez
shasaf1986,2019-05-30/2019-06-06
daikini
GianlucaCesari
AgentFabulous
josesm82
lmckenzie
