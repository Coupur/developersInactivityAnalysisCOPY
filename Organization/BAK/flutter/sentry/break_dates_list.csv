yjbanov,2017-08-07/2017-08-15,2017-08-15/2017-09-18,2017-09-18/2018-03-20,2018-03-21/2018-04-26,2018-04-26/2018-05-15,2018-05-15/2018-07-30,2018-07-30/2018-10-09,2018-10-09/2019-01-09
zoechi,2018-02-16/2018-02-27,2018-02-27/2018-03-02
pulyaevskiy
slightfoot,2018-04-26/2018-07-30
dustin-graham
dotdoom
filiph
kf6gpe,2018-10-29/2018-11-09
truongsinh
tom95
