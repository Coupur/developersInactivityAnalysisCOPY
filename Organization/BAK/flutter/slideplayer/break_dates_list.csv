0xflotus
KevinTheGray,2019-03-24/2019-03-28
