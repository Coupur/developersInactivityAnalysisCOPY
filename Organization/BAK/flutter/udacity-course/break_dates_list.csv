maryx,2017-09-18/2017-09-20,2017-09-20/2017-09-27,2017-09-27/2017-10-06,2017-10-06/2017-10-12,2017-10-13/2017-10-31,2017-10-31/2017-11-16,2017-11-17/2017-11-20,2017-11-20/2017-11-28,2017-11-28/2018-01-03,2018-01-03/2018-01-05,2018-01-05/2018-01-27,2018-01-27/2018-02-01,2018-02-01/2018-02-04,2018-02-04/2018-02-08,2018-02-08/2018-02-12,2018-02-13/2018-02-15,2018-02-15/2018-03-09,2018-03-09/2018-03-14,2018-03-15/2018-03-18,2018-03-18/2018-03-21,2018-03-21/2018-03-26,2018-03-26/2018-04-01,2018-04-02/2018-04-04,2018-04-05/2018-04-09,2018-04-09/2018-04-11,2018-04-11/2018-04-25,2018-04-27/2018-05-08,2018-05-08/2018-12-22,2018-12-22/2019-02-17
collinjackson,2017-11-13/2017-11-15,2017-11-16/2017-12-05,2017-12-05/2018-01-06,2018-01-06/2018-01-09
kroikie
mjohnsullivan,2017-11-17/2018-04-06
jonahwilliams
fdorssers
gaurav1999
apatil88
AdamMc331
robertgol
hexrcs
AskYous
SavinovJr
pw94
akashmehta
mit-mit
JDHatten
