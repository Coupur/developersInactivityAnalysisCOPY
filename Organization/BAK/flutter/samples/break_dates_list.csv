Hixie
RedBrogdon,2018-06-27/2018-06-29,2018-06-29/2018-07-02,2018-07-02/2018-07-20,2018-07-20/2018-07-26,2018-07-26/2018-07-30,2018-07-31/2018-08-08,2018-08-08/2018-08-31,2018-08-31/2018-09-05,2018-09-07/2018-09-13,2018-09-13/2018-11-08,2018-11-08/2018-12-06,2018-12-06/2018-12-10,2018-12-10/2018-12-12,2018-12-12/2018-12-14,2018-12-14/2018-12-19,2018-12-19/2018-12-21,2018-12-21/2019-01-03,2019-01-03/2019-02-27,2019-02-28/2019-03-13,2019-03-14/2019-03-18,2019-03-18/2019-03-20,2019-03-20/2019-03-22,2019-03-22/2019-03-25,2019-03-25/2019-05-05,2019-05-07/2019-06-10
filiph,2018-07-30/2019-01-02,2019-01-02/2019-05-16,2019-05-16/2019-05-20
mblaughton
willlarche,2018-09-27/2019-02-13,2019-02-13/2019-03-05
kenzieschmoll,2018-10-26/2018-11-01,2018-11-01/2018-11-09,2018-11-09/2018-11-13,2018-11-13/2019-03-20
roughike
burhanrashid52
amirh
ibhavikmakwana
rkinabhi
xster,2019-04-10/2019-04-12,2019-04-12/2019-06-10
timsneath
legalcodes
kevmoo,2019-05-02/2019-05-07,2019-05-08/2019-05-10,2019-05-10/2019-05-12,2019-05-12/2019-05-20,2019-05-20/2019-05-22,2019-05-22/2019-05-24,2019-05-25/2019-05-28,2019-05-31/2019-06-04,2019-06-04/2019-06-12
kford55
ptyagicodecamp
mit-mit
bratay,2019-06-12/2019-06-14,2019-06-14/2019-06-17
