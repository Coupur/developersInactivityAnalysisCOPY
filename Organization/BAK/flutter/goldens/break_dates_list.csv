tvolkert,2018-05-01/2018-05-14
Hixie,2018-05-08/2018-05-31,2018-06-01/2018-06-21,2018-06-21/2018-07-16,2018-07-16/2018-12-23,2018-12-23/2019-01-14,2019-01-14/2019-02-05
xster,2018-05-08/2018-05-10,2018-05-10/2018-06-01,2018-06-01/2018-06-05,2018-06-05/2018-06-14,2018-06-14/2018-07-03,2018-07-03/2018-12-19,2018-12-19/2019-02-20,2019-02-20/2019-04-24
jason-simmons
HansMuller,2018-06-05/2018-06-08
liyuqian,2018-06-28/2018-08-03,2018-08-04/2018-08-06,2018-08-07/2018-08-17,2018-08-17/2018-08-21,2018-08-22/2018-09-04,2018-09-04/2018-09-13,2018-09-13/2018-10-12,2018-10-12/2018-10-17,2018-10-17/2018-11-12,2018-11-12/2019-01-18,2019-01-18/2019-03-09,2019-03-09/2019-03-15,2019-03-15/2019-04-16,2019-04-16/2019-04-29
matthew-carroll
goderbauer,2018-07-03/2018-09-11,2018-09-12/2018-11-05,2018-11-05/2019-01-16
aam,2018-07-06/2018-07-10
cbracken,2018-07-11/2019-03-05
doxuanviet1996
amirh
nataliesampsell
mkustermann
jacob314,2018-08-29/2018-09-05
GaryQian,2018-09-07/2018-09-11,2018-09-12/2018-10-15,2018-10-15/2018-10-23,2018-10-24/2018-11-16,2018-11-16/2019-01-15,2019-01-15/2019-01-30,2019-01-31/2019-02-07,2019-02-07/2019-03-05,2019-03-05/2019-04-12,2019-04-12/2019-04-17,2019-04-17/2019-04-26,2019-04-27/2019-04-29,2019-04-29/2019-05-01,2019-05-01/2019-05-30,2019-05-30/2019-06-04
mklim
jonahwilliams
alexmarkov
johnsonmh,2018-09-19/2018-12-11,2018-12-11/2019-01-30,2019-01-30/2019-02-05,2019-02-05/2019-02-08,2019-02-08/2019-03-04
mraleph
beckler
Salby
dnfield,2019-01-10/2019-01-23,2019-01-23/2019-02-20,2019-02-20/2019-03-08,2019-03-08/2019-03-14,2019-03-14/2019-03-18,2019-03-19/2019-04-24
rami-a,2019-01-22/2019-01-24,2019-01-24/2019-01-30,2019-01-30/2019-03-04
jslavitz,2019-01-23/2019-01-29
shihaohong
LongCatIsLooong,2019-04-10/2019-06-01
justinmc
dkwingsmt,2019-05-29/2019-06-04
Piinks
